ftw/solr/__init__.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
 from zope.i18nmessageid import MessageFactory
 
+
 _ = MessageFactory('ftw.solr')
```

Comment:
i could not see any translations, drop the messagefactory?

Reply:
I've removed all translations related stuff as there are indeed no translations.